packages/cojson-transport-ws/src/index.ts

```diff
@@ -5,41 +5,19 @@ import {
     SyncMessage,
     cojsonInternals,
 } from "cojson";
+import { AnyWebSocket, PingMsg } from "./types.js";
 
-interface WebsocketEvents {
-    close: { code: number; reason: string };
-    message: { data: unknown };
-    open: void;
-}
-interface PingMsg {
-    time: number;
-    dc: string;
-}
-
-interface AnyWebSocket {
-    addEventListener<K extends keyof WebsocketEvents>(
-        type: K,
-        listener: (event: WebsocketEvents[K]) => void,
-        options?: { once: boolean },
-    ): void;
-    removeEventListener<K extends keyof WebsocketEvents>(
-        type: K,
-        listener: (event: WebsocketEvents[K]) => void,
-    ): void;
-    close(): void;
-    send(data: string): void;
-    readyState: number;
-    bufferedAmount: number;
-}
-
-const g: typeof globalThis & {
+ const g: typeof globalThis & {
     jazzPings?: {
         received: number;
         sent: number;
         dc: string;
     }[];
 } = globalThis;
 
+export const BUFFER_LIMIT = 100_000;
+export const BUFFER_LIMIT_POLLING_INTERVAL = 10;
```

Comment:
Lowered the value of the buffering limit to activate the backpressure system earlier.